src/nvim/os/os_defs.h

```diff
@@ -1,10 +1,140 @@
 #ifndef NVIM_OS_OS_DEFS_H
 #define NVIM_OS_OS_DEFS_H
 
+#include <ctype.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/types.h>
```

Comment:
Not going to work for MSVC. So why was this moved to os_defs.h, which is platform-agnostic?